compiler/surface/parser.messages

```diff
@@ -1741,7 +1741,7 @@ source_file: BEGIN_CODE SCOPE CONSTRUCTOR UNDER_CONDITION TRUE YEAR
 ## small_expression
 ##
 
-expected an enum constructor to test if the expression on the left
+expected the function application operator
```

Comment:
So the syntax has changed a lot and the update feature from menhir does not create the new error messages that would be needed with the new syntactic constructions. So from time to time I just re-create a new `parser.messages` from scratch with Menhir and git-merge it with the old one ; that way we keep the existing messages and add new ones for the new syntactic constructs. Maybe now is a good time to do that.

Reply:
Ah, I understand! Thanks for the hint.